example/README.md

```diff
@@ -218,8 +220,11 @@ If your tutorial depends on specific packages, simply add them to this provision
 
 * Dmitrii Tsybulevskii, 1st place of the [Yelp Restaurant Photo Classification](https://www.kaggle.com/c/yelp-restaurant-photo-classification). Link to [the Kaggle interview](http://blog.kaggle.com/2016/04/28/yelp-restaurant-photo-classification-winners-interview-1st-place-dmitrii-tsybulevskii/).
 
-## <a name="tools-with-mxnet"></a>Tools with MXnet
+## <a name="tools-with-mxnet"></a>Tools with MXNet
```

Comment:
What's the distinction now between this and the ecosystem page? Should we just more this whole section across.

Reply:
There's some duplication here. I could migrate these items to ecosystem. What do you think?